scripts/zones/Oldton_Movalpolos/mobs/Bugallug.lua

```diff
@@ -14,6 +14,16 @@ entity.onMobEngaged = function(mob)
 end
 
 entity.onMobDeath = function(mob, player, isKiller)
+    local party = player:getParty()
+    if #party > 1 then
+        for _, v in ipairs(party) do
+            if player:getZone() == v:getZone() and v:getQuestStatus(xi.quest.log_id.BASTOK, xi.quest.id.bastok.A_QUESTION_OF_FAITH) == QUEST_ACCEPTED then
+                v:setCharVar("Quest[1][77]Prog", 1)
```

Comment:
Same as last comment

Reply:
So for a greater question: Why is this not being handled in the quest script?  onMobDeath is supported in interaction.

Reply:
This is in the quest framework.
Are you saying a simple: quest:setVar(player, 'Prog', 1) sets for all players present in the zone upon death? or would I still need to check for zone and their individual prog